docs/README.md

```diff
@@ -21,8 +21,11 @@ except for versions lower than `.NET Framework 4.6.1`.
 
 ### Supported OSes
 
-OpenTelemetry .NET Auto-Instrumentation should work on all operating systems
-that given .NET version supports.
+The following operating systems are currently supported:
+
+- Windows,
+- MacOS,
+- Ubuntu.
 
```

Comment:
```suggestion

However, it should also work on other Linux OSes that given .NET version supports.

```

@RassK Checked that it also works e.g. on Alpine. Do you think that adding ☝️  makes sense?

Reply:
I think that a statement like that is useful.